Source/Public/Get-BicepConfig.ps1

```diff
@@ -0,0 +1,52 @@
+function Get-BicepConfig {
+    [CmdletBinding(DefaultParameterSetName='Default')]
+    param (
+        [Parameter(ParameterSetName='PathOnly', Mandatory)]
+        [Parameter(ParameterSetName='PathLocal', Mandatory)]
+        [Parameter(ParameterSetName='PathMerged', Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript( { Test-Path $_ } , ErrorMessage = "File not found")]
```

Comment:
Do we support folders? If not we should add ` -PathType Leaf` to test-path to only return true if it is a file.

Reply:
Yes we support folders, but if the folder and none of its parent folder contains a bicepconfig.json we will return an error stating "No valid configuration file found!"

Reply:
Thinking again, we will actually return the default config if no config is found. Get-BicepNetConfig returns the error above.